### ISSUE TITLE
New proposals 

### ISSUE THREAD

**Issue by davidsf** [openSUSE/osem#1899]
When CoP is open when a user click in Call of Papers first he goes to /proposals/index. If the CoP has a large description, when the user reach the end of the page and click "New proposal" (and goes to proposals/new) he sees "the same" page because the description is to large and the form if at the end of the page.   

Screencast about the issue:
![captura del escritorio de 16-12-17 19_49_08](https://user-images.githubusercontent.com/73978/34073465-d1ebde5c-e29a-11e7-962a-a4647149b28e.gif)

The option to solve the issue I think it could be to not show the description of the CoP in the new view. What do you think ?

**Comment by Ronaq13** [comment]
Yes, I think it will be a good user experience. What else I think is that we can put that description on new proposal page in a bootstrap collapse panel, so if the user wants to recheck the details then he can do that also without going back.
And yes I would like to work on this. :smile:

**Comment by differentreality** [comment]
For what it's worth I think you are misusing the field. Your cfp description has way too much information. For instance, you mention cfp(?) dates, but that's already mentioned at the end  of that description (in english). And that's one single example.
This field is not supposed to include all the information you would put in an article.

I wouldn't go as far as to limit the total number of words you can enter there, that's the organizers' responsibility. 

I do think it is potentially useful to show a **short** description on top of the submission form. I doubt people will pay attention to a link/button for information they are not actively looking for, but we (organizers) want to pass along.

**Comment by davidsf** [comment]
The use of this long information is for people that doesn't kwown the event I think is useful for the people that propose events.

I thinks osem should have this type of information in the call of papers, so I can share the direct link when announcing the CoP. 

The collapse panel should solve the issue for us.
